README.md

```diff
@@ -26,46 +26,44 @@ This build plugin is a utility for supporting Next.js on Netlify. To enable serv
 
 ## Installation and Configuration
 
-There are two ways to install the plugin in your Next.js on Netlify site: with the Netlify UI or with file-based installation.
+As of v3.0.0, all new sites deployed to Netlify with Next.js will automatically build with this plugin for a seamless experience deploying Next.js on Netlify!
 
-**UI-based Installation**
+This means that you don't have to do anything — just build and deploy your site to Netlify as usual and we'll handle the rest.
 
-You can install this plugin using this [direct in-app installation link](http://app.netlify.com/plugins/@netlify/plugin-nextjs/install) or from your team's [Plugins directory](https://app.netlify.com/plugins) in the Netlify UI.
+You're able to remove the plugin at any time by visiting the Plugins tab on your site.
 
-Read more about [UI-based plugin installation](https://docs.netlify.com/configure-builds/build-plugins/#ui-installation) in our docs.
```

Comment:
wait do we want to get rid of the UI install instructions flow for existing sites that wont get it auto-installed?

Reply:
great question. I may be misunderstanding how this works — for some reason I was under the impression that if you don't have any NoN plugins installed it would run it. @ehmicky @verythorough can y'all confirm?

Reply:
I am not too familiar with the UI auto-install logic unfortunately. 
@verythorough Do you know how this would happen?

Reply:
I was wrong — we need UI install for existing Next projects